docstr_coverage/printers.py

```diff
@@ -147,3 +164,63 @@ def _print_overall_statistics(self, results):
         )
 
         print_line("Total coverage: {:.1f}%  -  Grade: {}".format(count.coverage(), grade))
+
+
+class JsonPrinter(Printer):
+    def print(self, results):
+        """Prints a provided set of results to stdout in format JSON.
+
+        Parameters
+        ----------
+        results: ResultCollection
+            The information about docstr presence to be printed to stdout."""
+
+        file_results, total_results = results.to_legacy()
+        print(json.dumps({"files": file_results, "result": total_results}))
+
+
+class GitHubCommentPrinter(Printer):
```

Comment:
I am understanding it correctly that this is indeed Github specific (and not plain markdown) because of the icons you use, right? Just asking to double-check to understand why the class is named as it is. Maybe we should call it `GithubMarkdownPrinter`?

In either case, some docstring could not hurt ;-)